lib/ansible/modules/network/fortios/fortios_firewall_ipv6_eh_filter.py

```diff
@@ -44,76 +41,94 @@
     - fortiosapi>=0.9.8
 options:
     host:
-       description:
-            - FortiOS or FortiGate ip address.
-       required: true
+        description:
+            - FortiOS or FortiGate IP address.
+        type: str
+        required: false
     username:
         description:
             - FortiOS or FortiGate username.
-        required: true
+        type: str
+        required: false
     password:
         description:
             - FortiOS or FortiGate password.
+        type: str
         default: ""
     vdom:
         description:
             - Virtual domain, among those defined previously. A vdom is a
               virtual instance of the FortiGate that can be configured and
               used as a different unit.
+        type: str
         default: root
     https:
         description:
-            - Indicates if the requests towards FortiGate must use HTTPS
-              protocol
+            - Indicates if the requests towards FortiGate must use HTTPS protocol.
+        type: bool
+        default: true
+    ssl_verify:
+        description:
+            - Ensures FortiGate certificate must be verified by a proper CA.
         type: bool
-        default: false
+        default: true
+        version_added: 2.9
     firewall_ipv6_eh_filter:
         description:
             - Configure IPv6 extension header filter.
         default: null
+        type: dict
         suboptions:
             auth:
                 description:
                     - Enable/disable blocking packets with the Authentication header (default = disable).
+                type: str
                 choices:
                     - enable
                     - disable
-            dest-opt:
+            dest_opt:
                 description:
                     - Enable/disable blocking packets with Destination Options headers (default = disable).
+                type: str
                 choices:
                     - enable
                     - disable
             fragment:
                 description:
                     - Enable/disable blocking packets with the Fragment header (default = disable).
+                type: str
                 choices:
                     - enable
                     - disable
-            hdopt-type:
+            hdopt_type:
                 description:
                     - Block specific Hop-by-Hop and/or Destination Option types (max. 7 types, each between 0 and 255, default = 0).
-            hop-opt:
+                type: int
+            hop_opt:
                 description:
                     - Enable/disable blocking packets with the Hop-by-Hop Options header (default = disable).
+                type: str
                 choices:
                     - enable
                     - disable
-            no-next:
+            no_next:
                 description:
                     - Enable/disable blocking packets with the No Next header (default = disable)
```

Comment:
Doc description mentions "default = disable", then it should also be present in argspec and in doc too as `default: disable`.